admin.adoc

```diff
@@ -861,15 +861,12 @@ the following command line.
 grr_config_updater upload_python --file=myfile.py --platform=windows
 ------------------------------------------------------------------------
 
-At the end of this you should see something like:
-------------------------------------------------------------------------
-Uploaded successfully to aff4:/config/python_hacks/myfile.py
```

Comment:
Just checking - is this message not shown or do you think this information is of no use?